framework/Optimizers/GradientBasedOptimizer.py

```diff
@@ -216,16 +216,17 @@ def evaluateGradient(self, optVarsValues, traj):
       @ Out, gradient, dict, dictionary containing gradient estimation. gradient should have the form {varName: gradEstimation}
     """
     gradArray = {}
-    for var in self.optVars: gradArray[var] = np.ndarray((0,0))
+    for var in self.optVars:
+      gradArray[var] = np.ndarray((0,0)) #why are we initializing to this?
```

Comment:
So you would like me to revert this to same-line, or just to know that it will be removed?